exercises/rotational-cipher/rotational_cipher_test.exs

```diff
@@ -0,0 +1,80 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("rotational_cipher.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule RotationalCipherTest do
+  use ExUnit.Case
+
+  #@tag :pending
+  test "rotate a by 1" do
+    plaintext = "a"
+    shift = 1
+    assert RotationalCipher.rotate(plaintext, shift) == "b"
+  end
+
+  @tag :pending
+  test "rotate a by 26, same output as input" do
+    plaintext = "a"
+    shift = 26
+    assert RotationalCipher.rotate(plaintext, shift) == "a"
+  end
+
+  @tag :pending
+  test "rotate a by 0, same output as input" do
+    plaintext = "a"
+    shift = 26
```

Comment:
No wonder these tests looked so good and accurate - yay for macros! Might as well open a PR to fix the canonical data as well if you like. Your call - I'd be happy to handle it if you don't want the bother.